the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+            
+            // Configure the cell's switch state depending on wether the user is connected or not
+            if MyTBA.shared.authentication != nil {
+                cell.switchToggle.isOn = true
+            } else {
+                cell.switchToggle.isOn = false
+            }
+            
+            return cell
+            
+        case 1:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath) as! SwitchTableViewCell
+            cell.populate(withTitle: "Enable notifications")
+            
+            return cell
+
+        case 2:
+            cell.textLabel?.text = thirdSectionLabels[indexPath.row]
+            
+        case 3:
+            cell.textLabel?.text = lastSectionLabels[indexPath.row]
+            
+            cell.textLabel?.textColor = .red
+            cell.textLabel?.textAlignment = .center
+            
+        default:
+            fatalError("This row does not exist")
+        }
+        
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.section {
+        case 0:
+            // Handle authentification/unauthentification stuff
+            break
+            
+        case 1:
+            break
+            
+        case 2:
+            var urlString: String?
+            
+            switch indexPath.row {
+            case 0:
+                urlString = "https://www.thebluealliance.com"
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 1:
+                urlString = "https://github.com/the-blue-alliance/the-blue-alliance-ios"
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 2:
+                // TODO: Make a contributors list on Github
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 3:
+                // TODO: Make a changelog
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            default:
+                fatalError("This row does not exist")
+            }
+            
+            if let urlString = urlString {
+                let url = URL(string: urlString)
+                if let url = url, UIApplication.shared.canOpenURL(url) {
+                    UIApplication.shared.open(url, options: [:], completionHandler: nil)
+                }
+            }
+            
+        case 3:
+            if indexPath.row == 0 {
+                let alertController = UIAlertController(title: "Delete network cache", message: "Are you sure you want to delete all the network cache data ? This action is irreversible.", preferredStyle: .alert)
+                
+                let confirmAction = UIAlertAction(title: "Delete", style: .destructive) { (deleteAction) in
```

Comment:
Same here on the variable name - a deleteCacheAction or something seems like a better name